target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4018-magic-2-wifi-next.dts

```diff
@@ -0,0 +1,272 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qcom-ipq4019.dtsi"
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	model = "devolo Magic 2 WiFi next";
+	compatible = "devolo,magic-2-wifi-next";
+
+	memory {
+		device_type = "memory";
+		reg = <0x80000000 0x10000000>;
+	};
+
+	soc {
+		rng@22000 {
+			status = "okay";
+		};
+
+		mdio@90000 {
+			status = "okay";
+			pinctrl-0 = <&mdio_pins>;
+			pinctrl-names = "default";
+			reset-gpios = <&tlmm 59 GPIO_ACTIVE_LOW>;
+			reset-delay-us = <2000>;
+
+			/delete-node/ ethernet-phy@0;
+			/delete-node/ ethernet-phy@1;
+		};
+
+		ess-psgmii@98000 {
+			status = "okay";
+		};
+
+		crypto@8e3a000 {
+			status = "okay";
+		};
+
+		watchdog@b017000 {
+			status = "okay";
+		};
+
+		ess-switch@c000000 {
+			status = "okay";
+			switch_lan_bmp = <0x3e>;
+			switch_wan_bmp = <0x0>;
+		};
+
+		edma@c080000 {
+			status = "okay";
+			qcom,num_gmac = <3>;
+
+			gmac0 {
+				qcom,phy_mdio_addr = <3>;
+				qcom,poll_required = <1>;
+				qcom,poll_required_dynamic = <1>;
```

Comment:
Does this even exist in OpenWrt ESSEDMA driver?

Reply:
I looked at others (e.g. jalapeno) that had it but you are right, the driver doesn't recognize it. I've removed them.

Reply:
Do you maybe know what is that flag supposed to do?
I presume its in QCA-s version of the driver and was added later than OpenWrt version

Reply:
It's mentioned in a CAF submission:

https://patchwork.ozlabs.org/project/lede/patch/1492096230-20720-1-git-send-email-rjangir@codeaurora.org/

However it's not clear to me from the code what the intended function is.

Reply:
I looked at the QCA driver and it exists there, but later they simply set poll_required to poll_required_dynamic
So no idea what its supposed to do